packages/botonic-core/src/handoff.js

```diff
@@ -152,7 +152,6 @@ export async function getAvailableAgentsByQueue(session, queueId) {
     },
     method: 'post',
     url: `${baseUrl}/v1/queues/${queueId}/get_available_agents/`,
-    data: { queue_id: queueId },
   })
```

Comment:
I guess line removed by mistake
@guinii reviewers should stop these issues!

Reply:
@dpinol No, it's not removed by mistake and @guinii was aware of this line removed. We saw doing tests with Postman that the body in this request is not needed (in fact, the queueId is already in the URL itself!) and that's why I removed it

Reply:
@asastre @guinii my sincere apologies!

Reply:
@dpinol no problem! It's good that you see these (sometimes right and sometimes wrong) things :eyes:!!